src/main/java/fr/insee/pearljam/domain/campaign/model/communication/CommunicationTemplate.java

```diff
@@ -0,0 +1,18 @@
+package fr.insee.pearljam.domain.campaign.model.communication;
+
+
+/**
+ * A class representing the communication template of a visibility.
+ *
+ * @param id           The unique identifier of the communication template
+ * @param meshuggahId  The identifier for Messhugah
+ * @param medium       The medium of communication
+ * @param type         The type of communication
+ */
+public record CommunicationTemplate(
+        Long id,
```

Comment:
CommuncationTemplateId is missing => needed for UI

Reply:
id == communicationTemplateId